openpype/tools/utils/views.py

```diff
@@ -57,3 +59,68 @@ def paintEvent(self, event):
             self.paint_empty(event)
         else:
             super(TreeViewSpinner, self).paintEvent(event)
+
+
+class TreeView(QtWidgets.QTreeView):
```

Comment:
Should this also get a better class name now? Seems overly generic for something that only adds the capability to be "flickable"?

It does add double click signals too though.

Reply:
And also contains logic of `DeselectableTreeView`.